fastlane_bot/tools/invariants/functions/funcs.py

```diff
@@ -0,0 +1,81 @@
+"""
+functions library -- example functions
+
+(c) Copyright Bprotocol foundation 2024. 
+Licensed under MIT
+"""
+#__VERSION__    ==> core.__VERSION__
+#__DATE__       ==> core.__DATE__
+
+from .core import Function as _Function, dataclass as _dataclass
+import math as _m
+
```

Comment:
You might wanna use, for example, `a: float = 0.0` throughout this file.
Otherwise the declared variable is of type `int` regardless of the `float` hint.